completions/_git-forgit

```diff
@@ -1,12 +1,10 @@
-#compdef git-forgit
+#compdef git-forgit forgit::add forgit::branch::delete forgit::checkout::branch forgit::checkout::commit forgit::checkout::file forgit::checkout::tag forgit::cherry::pick forgit::cherry::pick::from::branch forgit::clean forgit::diff forgit::fixup forgit::log forgit::rebase forgit::reset::head forgit::revert::commit forgit::stash::show
```

Comment:
By listing all of these functions, this file will wake up and run on any of these commands, including aliases